src/ReverseProxy/Forwarder/ForwarderHttpClientFactory.cs

```diff
@@ -45,7 +46,10 @@ public HttpMessageInvoker CreateClient(ForwarderHttpClientContext context)
             UseProxy = false,
             AllowAutoRedirect = false,
             AutomaticDecompression = DecompressionMethods.None,
-            UseCookies = false
+            UseCookies = false,
+#if NET6_0_OR_GREATER
+            ActivityHeadersPropagator = Propagator
```

Comment:
```suggestion
            ActivityHeadersPropagator = new ReverseProxyPropagator(DistributedContextPropagator.Current)
```

We should still pick up the `DistributedContextPropagator.Current` over `CreateDefaultPropagator`.

I also don't think we need a property on `ForwarderHttpClientFactory` for this as the user can already change the propagator inside `ConfigureHandler`.